bin/cgp.sh

```diff
@@ -8,7 +8,7 @@
 #               -p <phase_number>               -- phase number of the upgrade (1)
 #               -s                              -- simulate the proposal (optional)
 #               -f                              -- use forked network (optional)
-# Example: ./bin/cgp.sh -n baklava -u MU01 -p 1 
+# Example: ./bin/cgp.sh -n baklava -u MU01 -p 1 || yarn cgp -n baklava -u MU01 -p 1_testnet_patch
```

Comment:
I have a couple of thoughts on this. 
-  @ninocomputer With this naming, the simulation script will not work as it expects the phase to be a uint8
- @bowd I feel this could get messy in future especially if we were to require multiple patches in one phase. e.g. another issue is discovered after the deployment of a patch. Should we switch to something like SemVer going forward? So CGP scripts for a future MU could look like this -> MU02-CGP-1-0-1

Reply:
Maybe, tho just pure semver might not be enough. For example this script is only meant to be run on testnets so we would need to differentiate somehow, or have some minor versions that do nothing in production environments or something weird like that. 

Initially I thought of combining this with the other `Phase1-patch1` script and have a conditional in the `prepare()` to add some transactions to the CGP only on testnets, but I thought that might be ugly and mixing up some testnet-only logic with production logic, so decided having two separate ones made sense. but yeah maybe we can do something like `semver + network modifier`. 

Let's revisit tho as we learn more about what we actually need to do in these scenarios, for now I think it's fine to keep it simple.